admetricks/slay
# feat(token): Allow Users to paste their tokens

## Allow setting API access tokens
Allow users to paste their tokens to get real, working code snippets

### Changelog
- Add token input
- Add util method to replace text in whole page (e.g. "YOUR_TOKEN" -> user token)
- Add 7-1 file structure for custom styles ([reference](https://gist.github.com/rveitch/84cea9650092119527bc))

### How to test
- Checkout this branch
- Run Slate locally (no stage env. yet) `sh ./run-dev.sh`
- Copy your Admetricks token and paste it in the **Token** form (right below "API")
- Copy some snippet and run it. It should work as-is

### Screenshots

No token

![image](https://user-images.githubusercontent.com/8508658/99966164-ad301d80-2d74-11eb-8fd1-7094ad9eaafa.png)


User pasted token

![image](https://user-images.githubusercontent.com/8508658/99966112-9e496b00-2d74-11eb-9511-b39478cb3e33.png)
